code/__defines/materials.dm

```diff
@@ -47,6 +47,12 @@
 #define MAT_VALUE_VERY_SHINY        60
 #define MAT_VALUE_MIRRORED          80
 
+// Paint flags
+#define PAINT_PAINTABLE 1
```

Comment:
There's a `BITFLAG` macro used for bitflags now. `BITFLAG(n)`, starting at 0 for `1`.